var/spack/repos/builtin/packages/libhio/package.py

```diff
@@ -24,6 +24,7 @@ class Libhio(AutotoolsPackage):
     # needed by current and future consumers of libhio
     #
     version('master', branch='master')
+    version('1.4.1.5', sha256='af5cb2a799a8470ed95847c3b07ea3ad61f8f7d5a2b79c52a46ca784846e8962', url="https://github.com/hpc/libhio/releases/download/1.4.1.5/libhio-1.4.1.5.tar.bz2")
```

Comment:
Two things:

* Could you close this and combine it with https://github.com/spack/spack/pull/23708?
* Either way, since you updated the `Package.url`, you should not have to define a version-specific URL here

Also a question: do the older versions download with the new URL format?